README.md

```diff
@@ -161,4 +161,4 @@ When a pull request is merged to master, it will automatically be deployed to th
 
 ### Production
 
-Coming soon...
+http://fightpandemics.com/
```

Comment:
Here. sorry, yeah we should use https and not http

Reply:
updated